joint_limits_interface/package.xml

```diff
@@ -0,0 +1,39 @@
+<package format="2">
```

Comment:
Why not use `format=3`

Reply:
The other packages are version 2, I can update this one.